hikari/impl/bot.py

```diff
@@ -448,15 +483,16 @@ def run(
         *,
         activity: typing.Optional[presences.Activity] = None,
         afk: bool = False,
+        idle_since: typing.Optional[datetime.datetime] = None,
+        status: presences.Status = presences.Status.ONLINE,
         asyncio_debug: typing.Optional[bool] = None,
         close_executor: bool = False,
```

Comment:
While we are here, can we rename this to `close_passed_executor` so that it is disambiguated from the default executor for the event loop, which is closed explicitly in >= CPython 3.9 only?

Reply:
Yep :)